description: fix incorrect calculation for the length argument in strncat
author: Michael Gilbert <mgilbert@debian.org>

--- a/tools/winedump/debug.c
+++ b/tools/winedump/debug.c
@@ -709,7 +709,7 @@ void    dump_stabs(const void* pv_stabs,
                 stabbufflen += 65536;
                 stabbuff = realloc(stabbuff, stabbufflen);
             }
-            strncat(stabbuff, ptr, len - 1);
+            strncat(stabbuff, ptr, stabbufflen - strlen(stabbuff) - 1);
             continue;
         }
         else if (stabbuff[0] != '\0')
